src/main/core-api/java/com/mysql/cj/conf/PropertyDefinitions.java

```diff
@@ -636,6 +636,9 @@
                 new StringPropertyDefinition(PropertyKey.clusterInstanceHostPattern, DEFAULT_VALUE_NULL_STRING, RUNTIME_NOT_MODIFIABLE,
                         Messages.getString("ConnectionProperties.clusterInstanceHostPattern"), "0.1.0", CATEGORY_HA, Integer.MIN_VALUE),
 
+                new BooleanPropertyDefinition(PropertyKey.useConnectionPlugins, DEFAULT_VALUE_TRUE, RUNTIME_NOT_MODIFIABLE,
```

Comment:
useConnectionPlugins -> enableConnectionPlugins

Reply:
This is to stay consistent with the PG naming. If we decide to use `enableConnectionPlugins` we need to change the option in PG as well @sergiyv-bitquill, thoughts?

Reply:
That's fine then. I'm okay with either